scripts/sync-plugin.mjs

```diff
@@ -0,0 +1,48 @@
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import {fileURLToPath} from 'node:url';
+
+const PROJECT_ROOT = path.dirname(fileURLToPath(import.meta.url));
+const ROOT_PKG_JSON_PATH = path.resolve(PROJECT_ROOT, '..', 'package.json');
+const PLUGIN_PKG_JSON_PATH = path.resolve(PROJECT_ROOT, '..', 'plugins', 'package.json');
+
+const SYNC_PACKAGE_KEYS = [
+  // To update ever version release
+  'version',
+
+  // These basic information should be the same with the top package.json
+  'engines',
+  'license',
+  'repository',
+  'author',
+  'bugs',
+  'homepage',
+];
+
+/**
+ * Return JSON parsed contents from the given path.
+ * @param {string} path
+ * @returns {Promise<object>}
+ */
+async function readJsonContent(jsonPath) {
+  return await JSON.parse(await fs.readFile(jsonPath, 'utf8'));
```

Comment:
JSON.parse does not return a promise, so await there is not needed

Reply:
oo, that's my bad in copy & paste somewhere